packages/datetime/src/dateRangeInput.tsx

```diff
@@ -739,8 +697,9 @@ export class DateRangeInput extends AbstractComponent<IDateRangeInputProps, IDat
         }) as DateRange;
     }
 
-    private getInputClasses = (boundary: DateRangeBoundary, boundaryInputProps: IInputGroupProps) => {
-        return classNames(boundaryInputProps.className, {
+    private getInputClasses = (boundary: DateRangeBoundary) => {
```

Comment:
same here, I think you can push this logic into `renderInputGroup`. I often use outline navigation in my editor to navigate to symbols so having lots of methods slows down that workflow